lib/tasks/ee.rake

```diff
@@ -0,0 +1,263 @@
+require 'yaml'
+require 'erb'
+
+class Database
+  RECOGNIZED_OPTIONS = %w(user host port socket)
+
+  def initialize(options = {})
```

Comment:
Can you use keyword args?

```ruby
def initialize(database:, **options)
  @database = database
  @options = options
end
```

This raises the error at the correct spot of `database` isn't provided